function-support.html

```diff
@@ -5,12 +5,12 @@
     <meta name="viewport" content="initial-scale=1">
     <title>Function Support in KaTeX</title>
     <link rel="icon" href="http://khan.github.io/favicon.ico">
+	
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/KaTeX/0.9.0/katex.min.css" integrity="sha384-TEMocfGvRuD1rIAacqrknm5BQZ7W7uWitoih+jMNFXQIbNl16bO8OZmylH/Vi/Ei" crossorigin="anonymous">

```

Comment:
We switched to `unpkg.com` for the main page, so presumably we should continue to follow suit here?

Reply:
I don't have a strong preference. Our READ.ME says to use the Cloudflare CDN, so I used it here to match. There have been a couple times when I have updated the function support page within one day of the release. I then use `unpkg.com` because the Cloudflare CDN can take 2 or 3 three days to propagate the files.

If you feel strongly about `unpkg.com`, I'll change it this time, too.

Reply:
I think it makes sense to match `index.html`, and that really does need quick updates (as it will coincide with release), so I'd strongly lean toward using `unpkg` here.